CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_zendesk_source v0.12.0
+[PR #49](https://github.com/fivetran/dbt_zendesk_source/pull/49) includes the following updates:
+## Breaking changes
+- Added field `_fivetran_deleted` to the following models for use downstream:
+  - `stg_zendesk__ticket`
+  - `stg_zendesk__ticket_comment`
+  - `stg_zendesk__user`
```

Comment:
Both `stg_zendesk__ticket` and `stg_zendesk__user` allow passthrough variables. We will want to include in these breaking change notes that if the `_fivetran_deleted` column is included in those variables then they will need to remove them.

Reply:
Updated!